### ISSUE TITLE
Offline mode is enabled while internet connection is fine

### ISSUE THREAD

**Issue by Mikeiscoola** [JiayiSoftware/JiayiLauncher#34]
everytime i launch the application as admin it says im offline but my wifi is on and great. what is happening? 
![image](https://github.com/JiayiSoftware/JiayiLauncher/assets/123332047/8f1a3673-c7a3-41df-8fb5-21f3cd407b9b)
when i have this speed of wifi 
![image](https://github.com/JiayiSoftware/JiayiLauncher/assets/123332047/5b9a3dce-3b79-49e5-a270-f6deb9b27a13)

**Comment by notcarlton** [comment]
can you go to 1.1.1.1 in a web browser

**Comment by Mikeiscoola** [comment]
> can you go to 1.1.1.1 in a web browser

yes. its the overriden thing for my wifi router from warp to my wifi

**Comment by notcarlton** [comment]
is offline mode on in the settings

**Comment by Mikeiscoola** [comment]
> is offline mode on in the settings

 i cant access settings. it crashes

**Comment by notcarlton** [comment]
send me the log file after it crashes

`%localappdata%\JiayiLauncher\Logs`

**Comment by notcarlton** [comment]
can you try the latest version and see if this issue was fixed?

if not, set your DNS servers to 1.1.1.1

**Comment by Mikeiscoola** [comment]
here!. i have this and thats all
[[CRASH] 2024-05-09_14-26-18.log](https://github.com/JiayiSoftware/JiayiLauncher/files/15262308/CRASH.2024-05-09_14-26-18.log)

**Comment by notcarlton** [comment]
did you do what I said after

**Comment by Mikeiscoola** [comment]
> did you do what I said after

how do i do that?
i can use winutils to do it right?

**Comment by notcarlton** [comment]
google it brother

**Comment by Mikeiscoola** [comment]
> google it brother

alt dns is 1.0.0.1?

**Comment by notcarlton** [comment]
yes

**Comment by Mikeiscoola** [comment]
ok

**Comment by Mikeiscoola** [comment]
done

**Comment by Mikeiscoola** [comment]
still offline mode.

**Comment by Mikeiscoola** [comment]
hey dude 
still broken
![image](https://github.com/JiayiSoftware/JiayiLauncher/assets/123332047/45219998-7e67-42ad-8939-a74b9ec3647d)

**Comment by notcarlton** [comment]
check ur firewall settings or smth idk

**Comment by Mikeiscoola** [comment]
![image](https://github.com/JiayiSoftware/JiayiLauncher/assets/123332047/1afbb7b3-4dc1-4cfd-976c-30d57cd64abe)

**Comment by TCK0423** [comment]
I have same problem :(

**Comment by TCK0423** [comment]
[[CRASH] 2024-08-14_15-31-42.log](https://github.com/user-attachments/files/16609142/CRASH.2024-08-14_15-31-42.log)

**Comment by phasephasephase** [comment]
but that could be our first clue, maybe your connection to the server jiayi pings to check for internet (1.1.1.1) is being blocked

**Comment by TCK0423** [comment]
> but that could be our first clue, maybe your connection to the server jiayi pings to check for internet (1.1.1.1) is being blocked

Bro, can u fix it?

**Comment by TCK0423** [comment]
> but that could be our first clue, maybe your connection to the server jiayi pings to check for internet (1.1.1.1) is being blocked

yesterday, it is normal.

**Comment by Renegade9152** [comment]
Alright, so I had the same problem: Every time I opened Jiayi Launcher, it would say "Offline Mode Enabled" 
I changed it in the settings to disable offline mode: Nothing happened. 
After I checked the Settings.json file, I found that on Line 20, "OfflineMode" was set to "true" Like this:

"OfflineMode": true,

So, I changed that to false, and it was all good. You can change it with Notepad that's built into Windows. 
SO, **To fix Offline Mode error**, Change line 20 to:
"OfflineMode": false,

**Comment by phasephasephase** [comment]
@Mikeiscoola @TCK0423 try this

**Comment by TCK0423** [comment]
> @Mikeiscoola @TCK0423 try this

it's already false.
but still ...

**Comment by TCK0423** [comment]
> @Mikeiscoola @TCK0423 try this

I have to use vpn, or it is still offlinemode

**Comment by phasephasephase** [comment]
then it's your internet/ISP at fault

**Comment by TCK0423** [comment]
> then it's your internet/ISP at fault

No, my internet is normal

**Comment by TCK0423** [comment]
> then it's your internet/ISP at fault

error code

System.Net.Sockets.SocketException (10060)

**Comment by TCK0423** [comment]
> then it's your internet/ISP at fault

[Current.log](https://github.com/user-attachments/files/17266881/Current.log)

**Comment by TCK0423** [comment]
my stats seems something problem ?

**Comment by phasephasephase** [comment]
most likely fixed in 2.1.0, try it

**Comment by Mikeiscoola** [comment]
### **Closing this issue.**

This Issue got fixed after i reset my pc in 2025/2/17, if it doesn't work. make your own issue
thank you. the creator. and thanks to everyone who tried to help me. pleasure having this open.
